Lesson5.md

```diff
@@ -2,7 +2,58 @@
 
 [Video](https://youtu.be/uQtTwhpv7Ew) / [Lesson Forum](https://forums.fast.ai/t/lesson-5-official-resources-and-updates/30863) 
 
-Welcome everybody to lesson 5. And so we have officially peaked, and everything is down hill here from here as of halfway through the last lesson. 
+Welcome everybody to lesson 5. And so we have officially peaked, and everything is down hill here from here as of halfway through the last lessclass TestAPIRegistry:
+    "Tests register which API they validate using this class."
```

Comment:
Did these get copied and pasted by mistake?

Reply:
oh probably - I guess I did that? Sorry about that !